Thingamajig.ts

```diff
@@ -0,0 +1,86 @@
+type Dimensions = {
+  width: number;
+  depth: number;
+  height: number;
+};
+interface IThingamajigHandler {
+  numberHandler: (t: Thingamajig) => void;
+  stringHandler: (t: Thingamajig) => void;
+  objectHandler: (t: Thingamajig) => void;
+}
+enum ThingamajigColor {
+  Red = "RED",
+  Blue = "BLUE",
+  White = "WHITE"
+}
+export default class Thingamajig {
+  name: string;
+  protected dimensions: Dimensions;
+  private label: any;
```

Comment:
S: The `any` type should be avoided whenever possible. In this case I think the following matches the intent:

```suggestion
  private label: "number" | "string" | "object";
```